File: benchmark/results/python-invocation-script/run-005/b-baseline/output/metaengine_demo/notification/services/delivery_dispatcher.py
"""DeliveryDispatcher for the notification domain."""
from typing import Optional

from metaengine_demo.notification.aggregates.notification import Notification


class DeliveryDispatcher:
    """DeliveryDispatcher."""

    def create(self, input: Optional[Notification]) -> Notification:
        raise NotImplementedError("not implemented")

    def find_by_id(self, id: str) -> Optional[Notification]:
        raise NotImplementedError("not implemented")

    def list(self, limit: float) -> list[Notification]:
        raise NotImplementedError("not implemented")

    def delete(self, id: str) -> None:
        raise NotImplementedError("not implemented")

File: benchmark/results/java-invocation-inline/run-002/b-baseline/output/com/metaengine/demo/notification/services/DeliveryDispatcher.java
package com.metaengine.demo.notification.services;

import java.util.List;

import com.metaengine.demo.notification.aggregates.Notification;

/** DeliveryDispatcher for the notification domain. */
public class DeliveryDispatcher {

    /** Create a new Notification from partial input. */
    public Notification create(Notification input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Find a Notification by id. Returns null if not found. */
    public Notification findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List Notifications up to the given limit. */
    public List<Notification> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Delete the Notification with the given id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}
